app/src/main/res/layout/downloads_topic_card.xml

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools">
+
+  <data>
+
+    <import type="android.view.View" />
+
+    <variable
+      name="isDeleteExpanded"
+      type="Boolean" />
+
+    <variable
+      name="viewModel"
+      type="org.oppia.android.app.mydownloads.downloads.DownloadsTopicViewModel" />
+  </data>
+
+  <com.google.android.material.card.MaterialCardView
+    android:id="@+id/download_container"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="28dp"
+    android:layout_marginTop="12dp"
```

Comment:
This should be `16dp`
BottomMargin should be removed as it won't work on root_layout.


If there is recyclerview and there is top-to-bottom hierarchy of views then prefer mentioning marginTop only.

Reply:
done